src/eureka/S3_data_reduction/source_pos.py

```diff
@@ -162,9 +164,20 @@ def source_pos(flux, meta, shdr, m, n, plot=True, guess=None):
         src_ypos, src_ywidth = source_pos_gauss(flux, meta, m, n, plot)
     elif meta.src_pos_type == 'hst':
         src_ypos = guess
-    else:
+    elif meta.src_pos_type == 'max':
         # brightest row for source location
         src_ypos = source_pos_median(flux, meta, m, n, plot)
+    elif not isinstance(meta.src_pos_type, str):
```

Comment:
No, the method doesn't work like that.  You're probably thinking of `meta.src_pos_type.isnumeric()`, but that only works when `meta.src_pos_type` is a string.  In our case, it can be a string or an integer (or a float, I guess), so that solution wouldn't work.